Source/VSProps/Base.props

```diff
@@ -95,6 +95,8 @@
       <AdditionalOptions>/Zo %(AdditionalOptions)</AdditionalOptions>
       <!--Treat sources as utf-8-->
       <AdditionalOptions>/utf-8 %(AdditionalOptions)</AdditionalOptions>
+      <!--Treat sources as utf-8-->
+      <AdditionalOptions>/utf-8 %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Aren't these two lines the same as the previous two lines?

Reply:
Indeed. Not sure how that snuck in there.